test/functionality/_helper/shell.ts

```diff
@@ -39,35 +39,14 @@ export const testWithShell = (msg: string, fn: (shell: RShell, test: Mocha.Conte
 	})
 }
 
-function installWarning(pkg: string) {
```

Comment:
teeechnisch gesehen scheitern jetzt die zugehörigen statistics tests oder? (in der Hoffnung das es sie gibt :P)